tests/__fixtures__/users/userReturns.fixtures.ts

```diff
@@ -0,0 +1,89 @@
+import type { UserReturns } from '../../../packages/client/src/users/returns/types';
+
+export const mockUserReturnsResponse: UserReturns = {
+  number: 1,
+  totalPages: 1,
+  totalItems: 1,
+  entries: [
+    {
+      id: 50057759,
+      orderId: 'YGG2Y8BK2',
+      merchantId: 9134,
+      userId: 30446182,
+      type: 'CourierPickUp',
```

Comment:
This file will have a lot of errors when type checking. Please, make sure the mock data is correctly defined respecting its type.